backend/src/test/java/kr/momo/domain/attendee/AttendeeEncryptedPasswordFixture.java

```diff
@@ -0,0 +1,22 @@
+package kr.momo.domain.attendee;
+
+import java.lang.reflect.Field;
+import org.springframework.security.crypto.argon2.Argon2PasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+public class AttendeeEncryptedPasswordFixture {
+
+    private AttendeeEncryptedPasswordFixture() {
+    }
+
+    private static final PasswordEncoder passwordEncoder = Argon2PasswordEncoder.defaultsForSpringSecurity_v5_8();
+
+    public static AttendeePassword createAttendeePassword(String rawPassword) throws Exception {
+        AttendeePassword attendeePassword = AttendeePassword.class.getDeclaredConstructor().newInstance();
```

Comment:
현재 AttendeePassword의 기본 생성자의 접근 제어자는 protected이기 때문에 아래와 같이 기본 생성자를 통해서 생성할 수 있습니다! 리플렉션으로 기본 생성자를 가져온 이유가 있나요?
```suggestion
        AttendeePassword attendeePassword = new AttendeePassword();
```

Reply:
오 배키처럼 구현하는게 더 간결하겠네요.
기존에 Fixture 패키지에서 리플렉션으로 시도해보았지만 protected 생성자의 경우 외부 패키지에서 리플렉션 사용이 안되더라구요. 
다만 최근에 protected 기본생성자여도 일부 설정을 통해 리플렉션으로 객체를 생성할 수 있음을 알게 되어 그 방식으로 수정 후 `EncryptedPasswordFixture`를 fixture 패키지로 옮겨둘게요.
덕분에 깨닫게 되었어요. 고마워요 배키 😁